setup.cfg

```diff
@@ -3,7 +3,7 @@
 exclude = .git,__pycache__,venv
 max-complexity = 10
 inline-quotes = '
```

Comment:
(-10P) Where is requirements.txt? No flake8 or flake8 modules are added.
-(5P) Where lost changes to the `setup.cfg`?